inference-engine/tests/functional/inference_engine/ngraph_reader/custom_op_tests.cpp

```diff
@@ -0,0 +1,243 @@
+            shape = castedNode->getShapeAttr();
+            data_ptr = castedNode->getDataPtr();
+            el_type = castedNode->get_input_element_type(0);
+        } catch (InferenceEngine::details::InferenceEngineException& ex) {
+            error = ex.what();
+            }
+        }
+
+        InferenceEngine::StatusCode
+        init(InferenceEngine::LayerConfig& /*config*/, InferenceEngine::ResponseDesc* /*resp*/) noexcept override {
+            return InferenceEngine::StatusCode::OK;
+        }
+
+        InferenceEngine::StatusCode getSupportedConfigurations(std::vector<InferenceEngine::LayerConfig>& conf,
+                                                               InferenceEngine::ResponseDesc* /*resp*/) noexcept override {
+            InferenceEngine::LayerConfig layerConfig;
+            layerConfig.dynBatchSupport = true;
+
+            if (node->outputs().size() != 1 && node->inputs().size() != 1)
+                return InferenceEngine::GENERAL_ERROR;
+
+            InferenceEngine::DataConfig cfg;
+            cfg.constant = false;
+            cfg.inPlace = 0;
+
+            InferenceEngine::SizeVector order;
+            auto partialShape = node->get_output_partial_shape(0);
+            if (partialShape.is_dynamic())
+                return InferenceEngine::GENERAL_ERROR;
+
+            auto shape = node->get_output_shape(0);
+            for (size_t i = 0; i < shape.size(); i++) {
+                order.push_back(i);
+            }
+            cfg.desc = InferenceEngine::TensorDesc(InferenceEngine::Precision::I32,
+                                                   shape, {shape, order});
+            layerConfig.outConfs.push_back(cfg);
+            layerConfig.inConfs.push_back(cfg);
+            conf.push_back(layerConfig);
+            return InferenceEngine::OK;
+        }
+
+        InferenceEngine::StatusCode
+        execute(std::vector<InferenceEngine::Blob::Ptr>& inputs, std::vector<InferenceEngine::Blob::Ptr>& outputs,
+                InferenceEngine::ResponseDesc* /*resp*/) noexcept override {
+            for (size_t i = 0; i < inputs.size(); i++) {
+                InferenceEngine::MemoryBlob::CPtr minput = InferenceEngine::as<InferenceEngine::MemoryBlob>(inputs[i]);
+                InferenceEngine::MemoryBlob::Ptr moutput = InferenceEngine::as<InferenceEngine::MemoryBlob>(outputs[i]);
+                if (!moutput || !minput) {
+                    return InferenceEngine::StatusCode::PARAMETER_MISMATCH;
+                }
+                // locked memory holder should be alive all time while access to its buffer happens
+                auto minputHolder = minput->rmap();
+                auto moutputHolder = moutput->wmap();
+
+                // retrieve constant data
+                const auto data_beg = static_cast<char*>(data_ptr);
+                const auto data_end = std::next(data_beg, shape_size(shape) * el_type.size());
+                std::vector<char> value;
```

Comment:
If vector reaction is required consider this:
```
std::vector<char> value{data_beg, data_end};
```
Or may be string?

P.S. I'm glad we don't have https://code-examples.net/en/q/2b9b0ab

Reply:
Corrected. String is not a proper type here as "value" is a byte-array and it is casted to int32.